cannon/main.go

```diff
@@ -23,17 +22,8 @@ func main() {
 		cmd.WitnessCommand,
 		cmd.RunCommand,
 	}
-	ctx, cancel := context.WithCancel(context.Background())
-
-	c := make(chan os.Signal, 1)
-	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
-	go func() {
-		for {
-			<-c
-			cancel()
-			fmt.Println("\r\nExiting...")
-		}
-	}()
+	// This used to punt stdout on an interrupt, should that be a default behaviour?
```

Comment:
It already logs below on an interrupt, I think the std-out addition in the go routine there isn't necessary. So this change is fine.

Reply:
Thanks